docs/man-pages/condor_token_create.rst

```diff
@@ -0,0 +1,123 @@
+      
+
+*condor_token_create*
+======================
+
+given a password file, create an authentication token
+:index:`condor_token_create<single: condor_token_create; HTCondor commands>`\ :index:`condor_token_create command`
+
+Synopsis
+--------
+
+**condor_token_create** **-identity** *user@domain* [**-keyid** *keyid*]
+[**-authz** *authz* ...] [**-lifetime** *value*]
+[**-token** *filename*] [**-debug**]
+
+**condor_token_create** [**-help** ]
+
+Description
+-----------
+
+*condor_token_create* will read a HTCondor password file inside the
+*SEC_PASSWORD_DIRECTORY* and use it to create an authentication token.
```

Comment:
Add "by default, this is the pool password file" or something to that effect; it's kind of buried otherwise.